lib/Service/NoteUtil.php

```diff
@@ -189,7 +189,7 @@ public function getNotesFolderUserPath(string $userId): ?string {
 		try {
 			$nodesFolder = $this->getOrCreateNotesFolder($userId, false);
 		} catch (NotesFolderException $e) {
-			$this->util->logger->warning("Failed to get notes folder for user $userId: " . $e->getMessage());
+			$this->util->logger->info("Failed to get notes folder for user $userId: " . $e->getMessage());
```

Comment:
```suggestion
			$this->util->logger->info("Failed to get notes folder for user $userId: " . $e->getMessage());
```

`info` is acceptable; `debug` better IMO since this isn't necessarily always a sign of any sort of problem

Reply:
Agreed in this case as we do not create it here automatically